proto/v1/issue_service.proto

```diff
@@ -318,7 +318,7 @@ message GrantRequest {
   // Format: roles/EXPORTER.
   string role = 1;
   // The user to be granted.
-  // Format: users/hello@bytebase.com.
+  // Format: users/{email}.
```

Comment:
Maybe we should follow just one unified format. Looks like we're using `users/{email}` for auth, database, logging, etc

Reply:
Yup, let's use "users/{email}".